wolfSSL/src/chapter07.md

```diff
@@ -100,10 +100,10 @@ extended key usage OID will cause a parsing error.
 #### Custom OID
     
 Custom OID injection and parsing was introduced in wolfSSL version 5.3.0. The 
-enable options `--enable-certgen` and `--enable-asn-template` must be used along 
-with the macros; `WOLFSSL_CUSTOM_OID` and `HAVE_OID_ENCODING` for working with 
-custom extensions. After building wolfSSL with these settings the function
-wc_SetCustomExtension can be used to set a custom extension in a Cert struct.
+enable options `--enable-certgen` must be used along with the macros
```

Comment:
I think it is important to also not the ASN template is required for this feature in case users are building with user_settings.h. Something like: "This feature only works with the newer ASN template code enabled by default with ./configure (`--enable-asn=template`) or manually if building with user_settings.h using `WOLFSSL_ASN_TEMPLATE`."

Reply:
I rechecked and was mistaken in https://github.com/wolfSSL/documentation/pull/94 regarding the `--enable-asn=template`. 

The setting with the equal sign does appear to be the correct option syntax.